src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/InterventionDTO.kt

```diff
@@ -5,13 +5,30 @@ import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.Interve
 data class InterventionDTO(
   val title: String,
   val description: String,
+  val pccRegions: List<String>,
+  val serviceCategory: String,
+  val serviceProvider: String,
```

Comment:
Do you think we could append `name` to these? (`serviceCategoryName`, `serviceProviderName`)

Reply:
Would the object work?
```json
"serviceCategory": {
  "id": ...,
  "name": ...,
}
```

I'm conscious we already have endpoints that return a representation of these concepts, so seems like we could reuse them. Is that valid?

Reply:
I am unsure the best way forward with these,  I am happy to either return as serviceCategoryName/serviceProviderName or as their own objects.
@Gweaton do you have a preference?

Reply:
Sorry @sldblog I'm a bit confused, are you saying we return the whole `serviceCategory` object that's returned from existing endpoints (e.g complexity level, desired outcomes too), or that we should just include `name` and `id` and use the `id` to make requests to the existing `service-category/{id}` endpoints?

Reply:
Just IDs would push the necessity of resolving those IDs to the client when displaying the list. That's also fine, but more complex on UI.

I was thinking of the full payload (embedding) because it's simpler for us in theory as we can reuse code that already exists and will be easier to document.

Feels like the complexity levels and desired outcomes could have their own endpoint (not for this PR though)

Reply:
Ah ok, that makes sense - shall we go for full payload then?

Reply:
how about `Summary` DTOs for a few of these entities. e.g. `ServiceCategorySummaryDTO` which are strict subsets of the full DTO. functionally this would be good to have for the interventions themselves too, where endpoints to query interventions return summaries, whereas endpoints to retrieve a specific intervention returns the full entity.

Reply:
If we are going to go for the summary approach, then I like having them as explicitly separate DTOs, instead of making a bunch of properties optional in the original DTO.

Reply:
After today's dev chat, we decided we'll use the full entities here for now.